src/pybind/mgr/dashboard/controllers/cluster.py

```diff
@@ -1,12 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from ..security import Scope
 from ..services.cluster import ClusterModel
-from . import APIDoc, APIRouter, EndpointDoc, RESTController
+from . import APIDoc, APIRouter, Endpoint, EndpointDoc, RESTController
 from ._version import APIVersion
 
 
-@APIRouter('/cluster', Scope.CONFIG_OPT)
+@APIRouter('/cluster')
```

Comment:
I had to remove the Scope permissions temporarly since I couldn't to get it working. This needs to be fixed before merging it.